doc/api/tls.md

```diff
@@ -1864,93 +1864,6 @@ The `cryptoStream.bytesWritten` property returns the total number of bytes
 written to the underlying socket *including* the bytes required for the
 implementation of the TLS protocol.
 
-### Class: `SecurePair`
```

Comment:
The entry in deprecations.md needs to be updated to reflect EOL status also

Reply:
How should it be updated? Couldn't find an example.

Reply:
![image](https://user-images.githubusercontent.com/439929/86949279-44b50c80-c103-11ea-81ce-a800c50e73be.png)

* Change `Type: Runtime` to `Type: End-of-life`
* Update the `changes:` under the YAML metadata.
* Update the text to reflect the fact that `createSecurePair()` has been removed.

Reply:
PTAL